frontend/src/features/recommendation/CriteriaFilter.tsx

```diff
@@ -49,14 +54,17 @@ const CustomSlider = withStyles({
     backgroundColor: 'currentColor',
   },
 })(Slider);
-
-interface Props {
+interface ValueLabelProps {
```

Comment:
minor: we can add a blank line for more clarity

```suggestion

interface ValueLabelProps {
```